actions/README.md

```diff
@@ -0,0 +1,48 @@
+# Trainer Actions
+
+The intent of the modules in this directory are to provide a fully detached way to
+achieve very specific tasks with Trainer. Notebooks are often tricky to get working
+for long running jobs due to various issues that can cause the kernel to die.
+
+The general goals of these actions are:
+
+- Purpose built Python modules that achieve a very specific task
+- Defining all inputs should be a function of environment variables i.e. with a `.env` file
+- The actions can be run in containers for portability
+
+# Container Build
+
+To build the container run:
+
+```
+docker build -t gretelai/trainer .
+```
+
+The `data` directory can be used as a volume mount into the container, it currently
+contains an example Gretel Config to be used with a demo DB.
+
+Within the container, we have a `/gretel/data` directory that should be used to mount
+to a directory on the host.
+
+# Example
+
+This example will take a single Gretel Transform Config and apply it to a demo MySQL DB
+running in Planet Scale and write the transformed data out to a SQLite db on disk.
+
+First create a `.env` file (in this directory) with the following contents, you'll need to change:
+
+- Gretel API Key
+- The password for the MySQL db
+
+```
+GRETEL_CONFIG=/gretel/data/transform_config.yml
+GRETEL_API_KEY=grtuXXX
+SOURCE_DB=mysql://dynf4plkdpg6e14abhf5:PASSWORD@aws.connect.psdb.cloud:3306/relational-demo?ssl=true&charset=utf8mb4
+SINK_DB=sqlite:////gretel/data/transformed.db
+```
```

Comment:
Should we add `.env` to the repo's gitignore just to be safe?

Also, is it worth checking in a `.env.example` file and instructing users here to `cp .env.example .env` and then adjust the newly-created file? I've always kind of liked that pattern.

Reply:
It's already in `.gitignore` all default Python ignore files from GitHub already contain that

Reply:
On the second note, yeah, I have a `.env.template` I didn't commit that has all the possible envs we support and was going to add some comments for how they are used. 

Arguably, any action could use any subset of all env vars depending on what's being done. For example Relational wouldn't make use of `GRETEL_PROJECT`, but a vanilla trainer based action might for just training a model + synthetic data output